proto/zitadel/user/v2beta/user_service.proto

```diff
@@ -238,6 +238,208 @@ service UserService {
     };
   }
 
+  rpc UpdateHumanUser(UpdateHumanUserRequest) returns (UpdateHumanUserResponse) {
+    option (google.api.http) = {
+      put: "/v2beta/users/{user_id}"
+    };
+
+    option (zitadel.protoc_gen_zitadel.v2.options) = {
+      auth_option: {
+        permission: "user.write"
+        org_field: "organization"
+      }
+      http_response: {
+        success_code: 201
+      }
+    };
+
+
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Delete user";
+      description: "The state of the user will be changed to 'deleted'. The user will not be able to log in anymore. Endpoints requesting this user will return an error 'User not found"
+      tags: "Users";
+      responses: {
+        key: "200"
+        value: {
+          description: "OK";
+        }
+      };
+    };
+  }
```

Comment:
Seems to be the wrong `openapiv2_operation` option. (is for Delete in an Update method)

Reply:
Corrected